packages/utils/interactive-hooks/src/Checkbox.hooks.ts

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+import { IKeyboardEvent } from '@office-iss/react-native-win32';
+
+export type onToggleCallback = (value: boolean) => void;
+export type boxSideType = 'start' | 'end' | undefined;
+export type keyUpCallback = () => void;
+export type checkedType = boolean | undefined;
+
```

Comment:
Prefer PascalCase for type names that aren't interface-like

Reply:
Got it - changing now